Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -98,6 +101,35 @@ const Contest = () => {
         [],
     );
 
+    const participantsStateText = useMemo(
+        () => isOfficial
+            ? 'Active'
+            : 'Total',
+        [ isOfficial ],
+    );
+
+    const participantsValue = useMemo(
+        () => isOfficial
+            ? activeParticipantsCount.toString()
+            : totalParticipantsCount.toString(),
```

Comment:
Is `strting` type necessary here? What's wrong with `number`?

Reply:
In my opinion, it is required because of the following error that appears: "Texts must have only text or children at Text".

Reply:
Can you try to remove it @PetarNeshkov? It seems unnecessary toString because the Text component can be used with ReactNode.